src/main/java/gameframework/base/ObservableValue.java

```diff
@@ -2,13 +2,31 @@
 
 import java.util.Observable;
 
+/**
+ * Class that represent an observable value.
```

Comment:
Useless: this is just paraphrasing the class name. Explain what's observed (value changes) and that this class actually plays the role of a variable rather than a value (look at its interface).